src/bundle/Resources/public/js/scripts/admin.limitation.pick.js

```diff
@@ -15,17 +15,33 @@
         event.preventDefault();
 
         const config = JSON.parse(event.currentTarget.dataset.udwConfig);
+        const confirmLabel = Translator.trans(
+            /*@Desc("Add locations")*/ 'subtree_limitation.confirm.label', {}, 'universal_discovery_widget');
+        const title = Translator.trans(
+            /*@Desc("Choose locations")*/ 'subtree_limitation.title',
+            {},
+            'universal_discovery_widget'
+        );
 
-        ReactDOM.render(React.createElement(global.eZ.modules.UniversalDiscovery, Object.assign({
-            onConfirm: selectLocationsConfirm.bind(this, event.target),
-            onCancel: closeUDW,
-            confirmLabel: 'Add locations',
-            title: 'Choose locations',
-            startingLocationId: window.eZ.adminUiConfig.universalDiscoveryWidget.startingLocationId,
-            multiple: true,
-            restInfo: {token, siteaccess}
-        }, config)), udwContainer);
+        ReactDOM.render(
+            React.createElement(
+                global.eZ.modules.UniversalDiscovery,
```

Comment:
If you follow my suggestion from the comment above, then `eZ.modules.UniversalDiscovery`.
Please update the following code accordingly.